support/llvm-project/cmake/caches/DistributionEspressif.cmake

```diff
@@ -0,0 +1,55 @@
+# This file sets up a CMakeCache for a simple distribution bootstrap build.
+
+#Enable LLVM projects and runtimes
+#set(LLVM_ENABLE_PROJECTS "clang" CACHE STRING "")
+set(LLVM_ENABLE_PROJECTS "clang;clang-tools-extra;lld" CACHE STRING "")
+set(LLVM_ENABLE_RUNTIMES "compiler-rt" CACHE STRING "")
+#set(LLVM_ENABLE_PROJECTS "clang;clang-tools-extra;lld" CACHE STRING "")
```

Comment:
Are these commented-out lines intensional here?

Reply:
espressif/llvm-project should release artifacts with clang in future also on GitHub. This part will be removed.